eng/versioning/version_client.txt

```diff
@@ -50,6 +50,7 @@ com.azure:azure-communication-common;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-communication-administration;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-sms;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-identity;1.0.0-beta.4;1.0.0-beta.5
+com.azure:azure-communication-phonenumbers;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-core;1.13.0;1.14.0-beta.1
```

Comment:
This file is updated after Feb. 9th release. Please rebase and correct the number. More than likely this line should read:

com.azure:azure-communication-phonenumbers;1.0.0-beta.5;1.0.0-beta.5